### PR TITLE
Cinema Wiki request

### PR DESCRIPTION
**Pull request by TheTVFan**

### CODE REVIEW

File: LocalSettings.php [hunk 1]
```diff
@@ -534,6 +536,15 @@
 				'consul',
 			),
 		),
+		'+cinemawiki' => array(
+			'bureaucrat' => array(
+				'editor',
+			),
+			'admin' => array(
```
Comment (Southparkfan) [inline LocalSettings.php]: admin should be sysop

File: LocalSettings.php [hunk 2]
```diff
@@ -534,6 +536,15 @@
 				'consul',
 			),
 		),
+		'+cinemawiki' => array(
+			'bureaucrat' => array(
+				'editor',
```
Comment (Southparkfan) [inline LocalSettings.php]: editor is not a valid user group

File: LocalSettings.php [hunk 3]
```diff
@@ -534,6 +536,15 @@
...
+			),
+			'admin' => array(
+				'bot',
+				'editor',
```
Comment (Southparkfan) [inline LocalSettings.php]: editor is not a valid user group

File: LocalSettings.php (hunk 2 above)
Comment (TheTVFan) [inline LocalSettings.php]: I would like to create the editor group, but I don't know how to do it

File: LocalSettings.php (hunk 1 above)
Comment (TheTVFan) [inline LocalSettings.php]: Corrected

File: LocalSettings.php (hunk 2 above)
Comment (Reception123) [inline LocalSettings.php]: @TheTVFan It would be easier if you come on IRC and then SPF can explain it to you.

**Comment by Reception123** [comment]
Conflicts. @Southparkfan @JohnFLewis please close

**Comment by NDKilla** [comment]
fixing in #134 with even more stuff so

**Pull request closed without merging**